src/helpers/issue-fetching.ts

```diff
@@ -167,15 +168,18 @@ export async function mergeCommentsAndFetchSpec(
 export async function fetchPullRequestDiff(context: Context, org: string, repo: string, issue: number): Promise<string | null> {
   const { octokit, logger } = context;
   try {
-    const { data } = await octokit.pulls.get({
-      owner: org,
-      repo,
-      pull_number: issue,
-      mediaType: {
-        format: "diff",
-      },
+    const githubDiff = new GithubDiff(octokit);
+    //Fetch the statistics of the pull request
+    const stats = await githubDiff.getPullRequestStats(org, repo, issue);
+    //Find the filenames which do not have more than 200 changes
+    let files = stats.filter((file) => file.changes < 500).map((file) => file.filename);
+    //Ignore files like in dist or build or .lock files
+    const ignoredFiles = ["dist/*", "build/*", ".lock", "index.js"];
```

Comment:
Should definitely be dynamic from git ignore and such. Otherwise this was not included in the spec.

Reply:
Does it combine this + gitignore?